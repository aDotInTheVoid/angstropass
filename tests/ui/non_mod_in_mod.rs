#[nanopass::langs]
mod foo {
    struct Bar;
    struct Baz;
}

fn main() {}
