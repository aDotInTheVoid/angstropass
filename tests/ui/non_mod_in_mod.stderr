error: expected a module
 --> tests/ui/non_mod_in_mod.rs:3:5
  |
3 |     struct Bar;
  |     ^^^^^^^^^^^
